### DISCUSSION TITLE
PorousFlow - Barometric pp

### DISCUSSION THREAD

**Discussion by MatiasAllay** [idaholab/moose#18678] [Q&A Modules: General] [answered]
Hi MOOSE,

I was testing the `barometric_fully_confined.i` and I got a question. Why when I increase the bulk modulus (lower compressibility)  the porepressure goes down? I would expect the opposite behavior since a harder material should make the fluid bear all the pressure change than a soft one. I'm confused.
Cheers!
Matias

**Comment by WilkAndy** [comment, marked as answer]
When K is increased the Skempton coefficient, B, decreases, so the porepressure responds less to external pressures (see the Doan reference for the derivation).  What is happening is that the stiff matrix is bearing most of the load, and the fluid doesn't have to bear much at all.  Remember this "fully confined" situation is rather contrived - the aquitards sitting above and below the model don't let the fluid "see" the external pressure explicitly, only through the mechanical deformations.

**Reply by MatiasAllay** [reply]
That answers my question. Thanks Andy
